content/docs/studio/user-guide/model-registry/add-a-model.md

```diff
@@ -38,23 +38,38 @@ https://www.youtube.com/watch?v=szzv4ZXmYAs
      [`mlem init`](https://mlem.ai/doc/command-reference/init), Iterative Studio
      will be able to parse the `.mlem` file to extract model metadata.
 
-4. Provide labels for your model. For example, if your model is about reviewing
+4. If the model is located on the remote storage, Iterative Studio will detect
+   it and suggest to enter path in the repository where to save the model file.
+   In this case, the following will happen:
+
+   - If the repository does not contain DVC, Iterative Studio will run
+     `dvc init`. It is needed to version the model in the git repository.
+     [Learn more](/doc/command-reference/init).
+   - If the specified directory does not exist yet, it will be created.
+   - Iterative Studio will import the model to the repository by executing
+     `dvc import-url <remote_path> <directory_path>/<filename from remote_path> --no-exec`.
+   - Iterative Studio annotate the model by executing
+     `gto annotate <model_name> --path <directory_path>/<filename from remote_path> --type model`.
+     [Learn more](https://github.com/iterative/gto#annotating).
+   - Iterative Studio will create a commit and push it to the repository.
```

Comment:
If I understand this correctly, this is supposed to make things easier (LW=lightweight), so should the docs get shorter instead of longer? I would at least try to combine this with the previous point, since it's about the same step, I think.

Also, try to write this from the user perspective a little more. Currently it seems like a description of the implementation (if this, then that...) Many details can be skipped if they're explained in the UI itself (which should be intuitive). Example (quick idea):

```
3. Enter the location of the model file. It can be a remote URL (S3, GDrive, etc.)
   or a local path (Git, DVC, MLEM).

   Models from remote URLs will be [imported](/doc/command-reference/impor-url)
   into a [new DVC project](/doc/command-reference/init) automatically and
   [annotated with GTO](https://github.com/iterative/gto#annotating).
```

Reply:
LW (lightweight) was not added in this iteration. Adding models was always LW 😄 
What we added this time is LW "with dvc". So we do more than before (we init dvc, create dvc pointer file for the model file, etc.). Therefore, more things to explain in the docs.

But agreed, we can revise this a bit more.

> Not sure we need to mention it, it's kind of required to save the new DVC+GTO repo (implementation detail).

I'd say we need to mention it. Coz otherwise users won't know where the model info is saved (in a database like many other model registries or in Git, which is unique to us).

Reply:
Got it, thanks. But still:

> we do more than before... Therefore, more things to explain in the docs.

This may is still thinking from the engineer's perspective. The code does more, but the product may actually be more straightforward as a result. Maybe it's not the case here but just something to keep in mind in general.

WDYT about my proposal to combine 3 and 4 ?

No strong opinion on mentioning what/when/where to commit. I just rec. to do it strategically (e.g. once for the whole doc).

Reply:
Updated.